### PR TITLE
 StackQueues.kt

### PR DESCRIPTION
**Pull request by Apollo9999**
StackQueues Algorithm Using Kotlin

### CODE REVIEW

File: Task 3/StackQueues [hunk 1]
```diff
@@ -0,0 +1,114 @@
+package stacksqueues
```
Comment (chandrikadeb7) [inline Task 3/StackQueues:1]: Rename file with .kt extension

Comment (Apollo9999) [reply Task 3/StackQueues:1]: Folder structure created for Stackqueues Using Kotlin provided.Approve the PR upon review

**Comment by Apollo9999** [comment]
Forked the Repo and Created all the stackqueues files in folder structure .Please let me know again do i need to create a new Pull request

**Comment by chandrikadeb7** [comment]
> Forked the Repo and Created all the stackqueues files in folder structure .Please let me know again do i need to create a new Pull request

Please make a new PR

**Pull request closed without merging**